src/main/java/com/netuitive/iris/client/AbstractRestClient.java

```diff
@@ -105,12 +105,10 @@ public AbstractRestClient() {
 
     private void setEntity(HttpEntityEnclosingRequestBase req, Object obj) {
         try {
-            req.setEntity(new StringEntity(mapper.writeValueAsString(obj)));
+            req.setEntity(new StringEntity(mapper.writeValueAsString(obj), "UTF-8"));
```

Comment:
Could you wrap this UTF-8 error and fix in a small test? Might mean pulling the `StringEntity` construction into a new function which works with `UTF-8` set and throws an exception when it it not.

Reply:
this is springbok server rejecting the request (400) without that explicit UTF-8 encoding which I verified locally. not sure if we can test this on the client side alone.

Reply:
Ok, I understand.